### PR TITLE
Cmake install full path

### PR DESCRIPTION
**Pull request by keerockl**
switching from CMAKE_INSTALL_LIBDIR into CMAKE_INSTALL_FULL_LIBDIR for lib full path for  51-smartx.conf.in

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ pkg_check_modules(DLT REQUIRED automotive-dlt-c++)
 add_subdirectory( deps/audio/common )
 
 string(REPLACE " -O3" " -O2" GPTP_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+string(REPLACE " -Werror" "" GPTP_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```
Comment (edersondisouza) [inline CMakeLists.txt:19]: This is unrelated to commit message - is it a leftover?

Comment (keerockl) [reply CMakeLists.txt:19]: yes please disregard I will resubmit.
